jaeger-core/src/main/java/io/jaegertracing/senders/HttpSender.java

```diff
@@ -43,47 +43,12 @@
    * http://localhost:14268/api/traces
    *
    * Uses the default {@link okhttp3.OkHttpClient} which uses {@link okhttp3.ConnectionPool#ConnectionPool()}.
-   * Use {@link HttpSender#HttpSender(java.lang.String, int, okhttp3.OkHttpClient)} to adjust parameters.
+   * Use {@link HttpSender.Builder} if you need to add more parameters
    */
   public HttpSender(String endpoint) {
```

Comment:
any reason to keep this ctor?

Reply:
Not really, It would be better to remove it to make the API more consistent. We can do 0.28.0 with more deprecations ASAP, then remove it